core/helpers/index.ts

```diff
@@ -1,10 +1,8 @@
-import rootStore from '@vue-storefront/core/store'
 import SearchQuery from '@vue-storefront/core/lib/search/searchQuery'
 import { remove as removeAccents } from 'remove-accents'
-import { Logger } from '@vue-storefront/core/lib/logger'
 import { formatCategoryLink } from '@vue-storefront/core/modules/url/helpers'
 import Vue from 'vue'
-import config from 'config'
+import config from 'config';
```

Comment:
```suggestion
import config from 'config'
```
i believe we would have linter problem here